compiler/rustc_passes/src/dead.rs

```diff
@@ -465,6 +465,17 @@ impl<'tcx> Visitor<'tcx> for MarkSymbolVisitor<'tcx> {
 
         self.in_pat = in_pat;
     }
+
+    fn visit_inline_const(&mut self, c: &'tcx hir::ConstBlock) {
```

Comment:
Do we still need the above `visit_anon_const` if const blocks should be handled in `visit_inline_const` instead?

Reply:
Yes. You could still want to mark `three` as live in:
```rust
match S::<3> {
    S::<{three()}> => {}
}
```